component/main.jsonnet

```diff
@@ -7,11 +7,57 @@ local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.vcluster;
 local instance = inv.parameters._instance;
+local ocpRoute = import 'ocp-route.libsonnet';
+local postSetup = import 'post-setup.libsonnet';
+
+local isOpenshift = std.startsWith(inv.parameters.facts.distribution, 'openshift') || inv.parameters.facts.distribution == 'oke';
+
+local sccRole = kube.Role('use-nonroot-v2') {
+  metadata+: {
+    namespace: params.namespace,
+  },
+  rules: [
+    {
+      apiGroups: [
+        'security.openshift.io',
+      ],
+      resourceNames: [
+        'nonroot-v2',
+      ],
+      resources: [
+        'securitycontextconstraints',
+      ],
+      verbs: [
+        'use',
+      ],
+    },
+  ],
+};
+local sccRoleBinding = kube.RoleBinding('vcluster-use-nonroot-v2') {
+  metadata+: {
+    annotations+: {
+      'vcluster.syn.tools/description': 'Allow vcluster to sync pods with arbitrary nonroot users by allowing the default ServiceAccount to use the nonroot-v2 scc',
+    },
+    namespace: params.namespace,
+  },
+  roleRef_: sccRole,
+  subjects: [
+    {
+      kind: 'ServiceAccount',
+      name: 'vc-%s' % instance,
+      namespace: params.namespace,
+    },
+  ],
+};
+
 
 // Define outputs below
 {
   '00_namespace': kube.Namespace(params.namespace) {
     metadata+: com.makeMergeable(params.namespaceMetadata),
   },
-  '10_cluster': cluster.Cluster(instance, params),
+  [if isOpenshift && params.ingress.enabled && params.ingress.host != null then '11_ocp_route']: ocpRoute.RouteCreateJob(instance, 'vc-%s' % instance, params.ingress.host),
```

Comment:
`RouteCreateJob` is a bit misnamed now, should be `RoutePatchJob` afaict

Reply:
Fixed